### PR TITLE
Undefined organisation

### PR DESCRIPTION
**Pull request by SimonLab**
check for org_id is equal to -1

### CODE REVIEW

File: lib/queries/people/edit.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ module.exports = function (userId, updatedProfile) {
       values.push(
         key
         + ' = '
-        + updatedProfile[key] === -1 ? 'null' : updatedProfile[key]);
+        + (updatedProfile[key] === -1 ? 'null' : updatedProfile[key])
```
Comment (jay-meister) [inline lib/queries/people/edit.js:13]: this seems dodgy!  But will work for now

**Review by jay-meister — APPROVED** [review]
okaaaay

**Review by mk4111 — APPROVED** [review]
lgtm! :bahamas:

File: lib/queries/people/edit.js (hunk 1 above)
Comment (SimonLab) [reply lib/queries/people/edit.js:13]: agree this will be updated as soon as we add validation on the fields with Joi

**Comment by SimonLab** [comment]
@mk4111 @JMurphyWeb thanks for the review :+1:

**Pull request merged**